shell/platform/darwin/ios/ios_external_view_embedder.mm

```diff
@@ -94,7 +96,11 @@
 
 // |ExternalViewEmbedder|
 bool IOSExternalViewEmbedder::SupportsDynamicThreadMerging() {
+#if FML_OS_IOS_SIMULATOR
```

Comment:
Is this also because iOS simulator supports the software backend?
```suggestion
// TODO(jonahwilliams): remove this once Software backend is removed for iOS Sim.
#if FML_OS_IOS_SIMULATOR
```

Reply:
Yup. The software backend can't split up rasterization and presentation like either the Skia GPU or Impeller backend can, so we can't unmerge the threads.

Reply:
Curious -- any reason we haven't removed the software backend for simulators yet? Maybe just haven't got round to it yet. I know it used to be default, and there was definitely a period where we didn't require a macOS release that required Metal, but I believe our macOS minimums are now at 10.14 Mojave, which requires Metal.

Reply:
the removal of the software backend is tied to whether or not g3 simulators have access to GPU passthrough. AFAIK this is still being rolled out.

Reply:
Ah makes sense. This hit us on GitHub as well -- it wasn't initially available with GitHub Actions, but they later enabled it.